client/my-sites/checkout/checkout-thank-you/index.jsx

```diff
@@ -473,6 +476,16 @@ export class CheckoutThankYou extends Component {
 					<PlanThankYouCard siteId={ this.props.selectedSite.ID } { ...planProps } />
 				</Main>
 			);
+		} else if ( domainRegistrationPurchase && selectedFeature === 'email-license' ) {
```

Comment:
@fredrikekelund Wondering if if it suffices to only check for `selectedFeature`, given it would only be set to `email-license` if users made the purchase via the `onboarding-with-email` flow, which guarantees a domain purchase (whether it be a new domain or a transferred domain that the user owns)

Reply:
`domainRegistrationPurchase` is there because we need to wait for the data to load. I suppose I could swap that for `this.isDataLoaded()` if that seems clearer?